src/View3d.ts

```diff
@@ -35,10 +36,21 @@ export interface View3dOptions {
   useWebGL2?: boolean;
 }
 
+const allGlobalLoadingOptions = {
+  numChunksToPrefetchAhead: 5,
+  prefetchAlongNonPlayingAxis: false,
+  throttleArrivingChannelData: true,
+};
+
 /**
  * @class
  */
 export class View3d {
+  // TODO because View3d is basically a top level entrypoint for volume-viewer,
+  // maybe it should create the VolumeLoaderContext with options passed in.
+  // (instead of having the loaderContext created externally)
+  public loaderContext?: VolumeLoaderContext;
```

Comment:
Makes me think that `View3d` maybe ought to have a `createLoader` method of its own that calls our existing `createLoader` function within this context. Gets closer to having `View3d` represent the "world" in which all these things exist, which seems to be a relatively common paradigm.

Reply:
I do like this idea.  Someday there might also be a View3d.createScene and getScene ... and that leads to thoughts like better separation of the "viewer" from the actual viewing panel/canvas